test/client-integration-test.js

```diff
@@ -23,16 +23,16 @@ import shopPoliciesFixture from '../fixtures/shop-policies-fixture';
 
 suite('client-integration-test', () => {
   const domain = 'client-integration-tests.myshopify.io';
-  const apiVersion = '2019-07';
-  const apiUrl = `https://${domain}/api/${apiVersion}/graphql`;
   const config = {
     storefrontAccessToken: 'abc123',
     domain
```

Comment:
Not directly related to your change but I couldn't help but notice this dangling reference. What is domain mapped to in this dictionary?

Reply:
It's shorthand for `domain: domain` so it has a value of `'client-integration-tests.myshopify.io'`

Reply:
Thanks. Just looked it up - _object literal property value shorthand_ - apparently it's a new syntactic sugar introduced in es6